src/insns/jalr_32bit.adoc

```diff
@@ -51,7 +51,7 @@ reported in the CAUSE field of <<mtval>> or <<stval>>:
 | CAUSE                 | pass:attributes,quotes[{cheri_int_mode_name}] | pass:attributes,quotes[{cheri_cap_mode_name}] | Reason
 | Tag violation         |      | ✔     | `cs1` has tag set to 0
 | Seal violation        |      | ✔     | `cs1` is sealed and the immediate is not 0
-| Permission violation  |      | ✔     | `cs1` does not grant <<x_perm>>
+| Permission violation  |      | ✔     | `cs1` does not grant <<x_perm>>, or the AP field could not have been produced by <<ACPERM>>
 | Invalid address violation  | ✔    | ✔     | The target address is invalid according to xref:section_invalid_addr_conv[xrefstyle=short]
 | Length violation      | ✔    | ✔     | Minimum length instruction is not within the target capability's bounds.
```

Comment:
Malformed check for capability pointer mode?

Reply:
that's included in the length violation case - but I've added it anyway